pkg/services/ngalert/api/tooling/definitions/cortex-ruler_test.go

```diff
@@ -89,10 +89,10 @@ func Test_Rule_Group_Marshaling(t *testing.T) {
 				Name:     "foo",
 				Interval: 0,
 				Rules: []PostableExtendedRuleNode{
-					PostableExtendedRuleNode{
+					{
```

Comment:
are these changes to this file (`cortex-ruler_test.go`) intentional? if not i recommend to remove these changes from the pull request.

Reply:
They are. I don't even know how that happened. Maybe one of my updates from master?

Reply:
I just compared this file to the one on master, and it matches exactly. I'm afraid that if I change this back, I might inadvertently revert something in master that is supposed to be there. I cannot find how to remove this file from my PR, so can we just leave this as-is?

Reply:
since your last merge-from-master it vanished fromt the diff, so this should be ok now.